config/locales/pl.yml

```diff
@@ -692,6 +694,8 @@ pl:
 
       <p>Tekst bazuje na <a href="https://github.com/discourse/discourse">polityce prywatności Discourse</a>.</p>
     title: Zasady korzystania i polityka prywatności %{instance}
+  themes:
+    default: Mastodon
```

Comment:
We actually don't need the themes map anymore; we don't use it and Travis will fuss at us if it's included I think.